extensions/amp-animation/0.1/web-animations.js

```diff
@@ -252,6 +246,17 @@ export class Builder {
     return new MeasureScanner(this, this.css_, path,
         target, index, vars, timing);
   }
+
+  /**
+   * @return {boolean} Whether animationWorklet can be used.
+   * @private
+   */
+  isAnimationWorkletSupported_() {
+    return isExperimentOn(this.win_, 'chrome-animation-worklet') &&
+    'animationWorklet' in CSS &&
+    getMode(this.win_).runtime != 'inabox' &&
+    !isInFie(this.win_.document.documentElement);
```

Comment:
@aghassemi I would double check this. 
Based on what I read `Builder` class is created with `ampdoc.win`. FIE and parent document share the same `ampdoc`, so I am afraid that the `this.win_` here is still the parent window, not the FIE window.

Reply:
Builder is created with the host window (`hostWin = this.embed_ ? this.embed_.win : this.win;`) so it ends up correct and the FIE win.

Reply:
Thank you Ali for confirming. 
I think I read this
https://github.com/ampproject/amphtml/blob/baaa93350b5c654e64cd87b3b14d10706c95f564/extensions/amp-animation/0.1/web-animation-service.js#L43-L48
Is the usage here correct? Thank you

Reply:
Good find. looks like that's `amp-story`-only. Normal case goes through: https://github.com/ampproject/amphtml/blob/master/extensions/amp-animation/0.1/amp-animation.js#L472 so we are good (there is no scrollbound animations in amp-story and amp-story is not allowed in A4A, I don't see either of these changing ever)

Reply:
Thanks for the explanation. That makes a lot of sense. Might worth leaving a comment there though.